apps/api/package.json

```diff
@@ -10,7 +10,9 @@
     "prisma-reset": "npx prisma migrate reset",
     "prisma-status": "npx prisma migrate status",
     "prisma-sync": "npx prisma db push",
-    "pscale-connect": "pscale connect biti dev --port 3309"
+    "pscale-connect": "pscale connect biti dev --port 3309",
+    "prisma-init": "npx prisma init",
+    "prisma-generate": "npx prisma generate"
   },
```

Comment:
This PR containts two lock files, namely `package-lock.json` and `yarn.lock`. It's advisable to have only one lock file per project. I recommend deleting `package-lock.json`.

Reply:
alright working on this, update will be seen in my next update.

Reply:
@Ladu-Lumori resolve the conversation after you're done.